docs/src/pages/TypesettingPage/TypesettingPage.js

```diff
@@ -68,18 +69,22 @@ const components = [
       android: (
         <section>
           <Paragraph>
-            When designing for Android, use <strong>Roboto</strong>.
+            When designing for Android, use{' '}
```

Comment:
`{' '}` - this might be accidental?

Reply:
prettier has added that to maintain the space.
If we write 
```
...for Android, use
<strong>Skyscanner...
```
Then it will render ...for Android, use**Skyscanner...**

The `{' '}` tells it to render a space before the next element, so it will be ...for Android, use **Skyscanner...**

Reply:
Hmm ok as it hasn't done that for the iOS and Web version 🤔

Reply:
Because the line length was such that it didn't need to split it. Must be the extra 4 letters in Android

Reply:
🤷‍♂ who knows